config/monitoring/tracing/jaeger/102-role.yaml

```diff
@@ -0,0 +1,87 @@
+# Copyright 2019 The Knative Authors
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     https://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+apiVersion: rbac.authorization.k8s.io/v1
+kind: ClusterRole
+metadata:
+  creationTimestamp: null
+  name: jaeger-operator
+  namespace: istio-system
+rules:
+- apiGroups:
+  - ""
+  resources:
+  - pods
+  - services
+  - endpoints
+  - persistentvolumeclaims
+  - events
+  - configmaps
+  - secrets
+  - serviceaccounts
```

Comment:
We use the array notation in `200-clusterrole.yaml` instead of the list notation. Does it make sense to use the same here?

Reply:
Only reason I would prefer to leave this way, is that it is a straight copy from the upstream operator, so will be easier to maintain.

Reply:
sgtm